plugins/org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/ui/AbstractAutoEditTest.java

```diff
@@ -37,6 +39,8 @@
 	public void setUp() throws Exception {
 		super.setUp();
 		closeWelcomePage();
+		IProject project = createProject("foo");
```

Comment:
they are all creating a 'foo' project, but there is some that don't check whether it exists. So this change actually breaks :-|. Moved the creation down to the concrete class